archive/rest-catalog/brands.v2.yaml

```diff
@@ -0,0 +1,515 @@
+            schema:
+              $ref: '#/components/schemas/BrandPost'
+          'application/xml':
+            schema:
+              $ref: '#/components/schemas/BrandPost'
+        required: true
+      responses:
+        '201':
+          $ref: '#/components/responses/brand'
+        '400':
+          $ref: '#/components/responses/errors'
+        '401':
+          $ref: '#/components/responses/errorsWithDetails'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+        '409':
+          $ref: '#/components/responses/errorsWithDetails'
+    delete:
+      tags:
+        - Brands V2
+      summary: Deletes all existing on store brands.
+      description: Deletes all existing on store brands.
+      operationId: deleteBrands
+      responses:
+        '204':
+          description: No content
+          content: { }
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+  '/brands/{brand_id}':
+    get:
+      tags:
+        - Brands V2
+      summary: Gets a brand.
+      description: Returns a single *Brand*.
+      operationId: getBrandById
+      responses:
+        '200':
+          $ref: '#/components/responses/brand'
+        '400':
+          $ref: '#/components/responses/errors'
+        '401':
+          $ref: '#/components/responses/errorsWithDetails'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+    put:
+      tags:
+        - Brands V2
+      summary: Updates an existing Brand.
+      operationId: updateBrand
+      description: |-
+        Updates a *Brand*.
+
+        **Required Fields**
+        - None
+
+        **Read-Only Fields**
+        - id
+
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BrandPut'
+        required: true
+      responses:
+        '200':
+          $ref: '#/components/responses/brand'
+        '400':
+          $ref: '#/components/responses/errors'
+        '401':
+          $ref: '#/components/responses/errorsWithDetails'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+    delete:
+      tags:
+        - Brands V2
+      summary: Deletes a Brand.
+      description: Deletes a *Brand*.
+      operationId: deleteBrand
+      responses:
+        '204':
+          description: No content
+          content: { }
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+    parameters:
+      - $ref: '#/components/parameters/BrandIdParam'
+  '/brands/count':
+    get:
+      tags:
+        - Brands V2
+      summary: Count Brands.
```

Comment:
```suggestion
      summary: Count Brands. (Deprecated)
```

Reply:
Changed to "summary: Count Brands. (Deprecated)"